research/src/pages/tabs.research.mdx

```diff
@@ -0,0 +1,19 @@
+---
+name: Tabs (Working Draft)
```

Comment:
I'm making a change to the nav structure to handle this type of scenario. Also, the research doesn't progress like a spec so it doesn't have a stage.

```
---
name: Tabs
menu: Analysis
path: /components/tabs.research
pathToProposal: /components/tabs
---
```